.github/workflows/release.yaml

```
@@ -0,0 +1,35 @@
+name: release
+on:
+  push:
+    tags:
+    - '*'
+
+jobs:
+  release:
+    name: Upload release to PyPI
+    runs-on: ubuntu-latest
+    if: github.repository == 'pytest-dev/pytest-timeout'
+
+    permissions:
+      id-token: write  # IMPORTANT: this permission is mandatory for trusted publishing
+
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: 3.7
+
+      - name: Install dependencies
+        run: |
+          python -m pip install -U pip
+          python -m pip install -U build pytest>=5.0.0
```

```suggestion
          python -m pip install -U build
```

pytest isn't needed to build